Translations/Language.zh-cn.xml

```diff
@@ -1607,8 +1607,8 @@
     <entry lang="zh-cn" key="EXPANDER_ERROR_MAX_FILE_SIZE_EXCEEDED">超过了外部卷最大 %I64u MB 的文件限制。</entry>
     <entry lang="zh-cn" key="EXPANDER_ERROR_QUICKEXPAND_PRIVILEGES">错误：无法获得启用快速扩展所需的权限！\n请取消选中“快速扩展”选项，然后重试。</entry>
     <entry lang="zh-cn" key="EXPANDER_ERROR_MAX_VC_VOLUME_SIZE_EXCEEDED">超过了VeraCrypt卷最大 %I64u TB的文件限制！\n</entry>
-    <entry lang="zh-cn" key="FULL_FORMAT">完全格式化</entry>
-    <entry lang="zh-cn" key="FAST_CREATE">快速创建</entry>
+    <entry lang="zh-cn" key="FULL_FORMAT">完全格式化(随机填充)</entry>
+    <entry lang="zh-cn" key="FAST_CREATE">快速创建(不填充)</entry>
```

Comment:
Like my comment in quick format, introducing terms like "random fill" and "no filling" without proper context may be ambiguous for users unfamiliar with the details of formatting. While we, as developers, comprehend the details and mechanics, many of VeraCrypt users may not. Such terminology in the UI, without further explanation, could be potentially misleading or puzzling.

A more user-centric approach might be to enhance the documentation with clear explanations of each formatting operation. We could then provide a link or incorporate a help icon within the UI, guiding users who wish to learn more about the specifics of each operation.

It's vital to always design our UI elements with all users in mind, especially those who might not have a technical background.

Reply:
> 赞同，所以我建议所有语言的翻译都修改；更容易理解。 完全格式化 ➡️ 安全格式化 快速格式化 ➡️ 简单格式化

The Chinese term '快速格式化 (Quick formatting)' has been widely used in Windows for decades, and its corresponding actual meaning has a strict directional effect from beginning to end (for professionals, they will know how it will work as they just seeing the term; for non-professionals, they can also expect it to play the same role as in Windows or even Android). Inventing new words to describe a mature concept that has already been used in the industry will only lead to broader misunderstandings.